typings/src/ui/ui-events.ts

```diff
@@ -36,10 +36,12 @@ export const enum NotificationEvents {
   UnsnoozeNotifications = 'unsnooze-notifications',
 }
 
+export type NotificationType = 'info' | 'alert' | 'caution' | 'success' | 'error';
```

Comment:
You can convert this to an enum and then import it anywhere it's needed instead of using hardcoded values